### DISCUSSION TITLE
Is there a way for setting up a node/express server on the same electron-vite app?

### DISCUSSION THREAD

**Discussion by otekdo** [alex8088/electron-vite#104] [Q&A] [answered]
Hello, first of all,  thanks for this awesome toolset
But I would like to know how i can set up  additionally a node/express webserver on the same electron-vite app?

Example: I really love how i have an easy react-electron-vite boilerplate, but additionally, 
i'm also trying to have a **node/express/socket.io webserver** ideally a **websocket** server. So how i can accomplish this?

**Comment by theogravity** [comment, marked as answer]
Your question IMO is out of scope for this project since the method to add this sort of support probably applies to any electron project.

Just search for `electron express` or `electron websocket server` and you'll get results that you want.

**Reply by otekdo** [reply]
Thanks!

**Comment by alex8088** [comment]
`node/express/socket.io webserver` can only be hosted in the main process.

**Reply by otekdo** [reply]
Thanks!
